### PR TITLE
[AMSDK-9547] Updating the release notes for Identity v2.2.1 and Core v2.5.1

### PR DESCRIPTION
**Pull request by samthiru**
[AMSDK-9547] Updating the release notes for Identity v2.2.1 and Core v2.5.1

### CODE REVIEW

**Comment by nporter-adbe** [comment]
Also, update the extension-specific release notes.
https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-release-notes

**Comment by samthiru** [comment]
> Also, update the extension-specific release notes.
> https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-release-notes

Done! Thanks!

File: release-notes.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+##  March 12, 2020
+
+The following updates were made in this release:
+
+### iOS Core 2.5.1
```
Comment (kevinlind) [inline release-notes.md]: You don't need to list the Identity changes under Core as well. Only the changes specific to Core should get listed under Core. It is sufficient to just list the Identity changes under Identity.

File: using-mobile-extensions/mobile-core/mobile-core-release-notes.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Release Notes
 
+## March 12, 2020
+
+The following updates were made in this release:
+
+### iOS Core 2.5.1
```
Comment (kevinlind) [inline using-mobile-extensions/mobile-core/mobile-core-release-notes.md]: Same as above. You do not need to copy the Identity changes under the Core heading.

**Pull request merged**